app/src/main/java/jp/co/soramitsu/app/root/navigation/Navigator.kt

```diff
@@ -786,7 +784,8 @@ class Navigator :
     }
 
     override fun finishSendFlow() {
-        navController?.navigate(R.id.finish_send_flow)
+        navController?.popBackStack()
```

Comment:
are you sure, we need to call it twice?

Reply:
one for confirm screen and one for main screen